site/content/docs/user/private-registries.md

```diff
@@ -63,19 +63,24 @@ An example bash snippet for doing this with with [gcr.io][GCR]:
 # login to GCR on all your kind nodes
 # KUBECONFIG should point to your kind cluster
 export KUBECONFIG="$(kind get kubeconfig-path --name="kind")"
+
+# https://cloud.google.com/container-registry/docs/advanced-authentication#access_token
+gcloud auth print-access-token | docker login -u oauth2accesstoken --password-stdin https://gcr.io
```

Comment:
this however can be disruptive to the host config.json, in case someone has manually set up their host already.
i think the old approach is better in that regard.

Reply:
As @chemikadze brought up in https://github.com/kubernetes-sigs/kind/commit/6b796849caec75dba1699a02d38cb67a0596d637, the old method will no longer work with containerd.

See https://github.com/kubernetes-sigs/kind/issues/594#issuecomment-507824471. More specifically, @BenTheElder's comments about a potential solution

> The docker login generated credentials just need to be generated by another means or on the host.

We can script around the disruption of the host config? (ie. move the old config out of the way and move it back later)

Reply:
> the old method will no longer work with containerd.

looks like i've missed the existing comments.

> We can script around the disruption of the host config? (ie. move the old config out of the way and move it back later)

backup on the existing configuration on the host, seems good to me.

Reply:
Sounds like a plan. I'll get those changes made soon :tm:

Reply:
Access tokens seem to expire in a matter of hours, does it make sense to use serviceaccount-based auth in the example?

Reply:
@chemikadze let's do that in a follow up, since this _works_ and the current documentation is broken. I agree though.